pipeline/packer/jenkins.yaml

```diff
@@ -0,0 +1,53 @@
+jenkins:
+  systemMessage: "Auto-configured integration-adaptors Jenkins instance.\n\n"
```

Comment:
TODO-could get rid of this, or change it to something more meaningful, as it's getting a bit annoying seeing it in Jenkins.